test-resources/xmir-parsing-packs/base-dispatch.yaml

```
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+xmir: |
+  <object>
+    <o name="app">
+      <o name="x" base="Q.org.eolang"/>
+    </o>
+  </object>
+phi: '{[[ app -> [[ x -> Q.org.eolang ]] ]]}'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)